### ISSUE TITLE
grep: Invalid content of \{\}

### ISSUE THREAD

**Issue by Stretch96** [OctoPrint/OctoPrint-FileCheck#2]
#### What were you doing?

Restarted OctoPrint after upgrading to 1.5.3

#### What happened?

Noticed an odd line in the logs:

```
Jan 28 21:44:46 raspberrypi octoprint[1111]: grep: Invalid content of \{\}
```

OctoPrint still runs fine, appears to be caught as just a warning, just raising as an issue 👍

**Comment by cp2004** [comment]
In case it is useful, I experimented with this the other day - still couldn't work out *why* it failed, but I had the same message in the stdout from running `octoprint serve` on my Pi (through remote debug).

```
pi@octopi:~ $ grep -q '^[^;]*\\{travel_speed\\}' '/home/pi/.octoprint/uploads/xyzCalibration_cube.gcode'
pi@octopi:~ $ grep -q '^[^;]*\{travel_speed\}' '/home/pi/.octoprint/uploads/xyzCalibration_cube.gcode'
grep: Invalid content of \{\}
```
Something's up with the escaping somewhere

**Comment by foosel** [comment]
Yeah, I figured it out. Grep by default doesn't need those brackets escaped because it runs in simple regex mode (where those don't have a special meaning). Adding `-E` solves it however. I'm prepping an update.

The sad thing is that for some reason my local grep on my Win10 box doesn't have this issue, since it only ever ended up in syslog I didn't notice it while testing on the Pi, and thus this whole check probably never so far ran in the field 🤦‍♀️

Ah well... adding fix & logging.

**Comment by foosel** [comment]
Fixed in 2021.2.23

**Comment by Stretch96** [comment]
Awesome, thanks @foosel @cp2004 for fixing 😬
